diff --git a/src/Makefile b/src/Makefile
index 44f43fe..25f00de 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -62,9 +62,9 @@ CXXFLAGS_OPTIMIZE ?= -O2
 endif
 CXXFLAGS += $(CXXFLAGS_OPTIMIZE)
 # protect against security threats caused by misguided C++ compiler "optimizations"
-ifeq ($(findstring clang,$(CXX)),)
-CXXFLAGS += -fno-delete-null-pointer-checks
-endif
+#ifeq ($(findstring clang,$(CXX)),)
+#CXXFLAGS += -fno-delete-null-pointer-checks
+#endif
 CXXFLAGS += -fno-strict-aliasing -fwrapv
 CXXFLAGS += -funsigned-char
 CXXFLAGS += -Wall -W -Wcast-align -Wcast-qual -Wmissing-declarations -Wpointer-arith -Wshadow -Wvla -Wwrite-strings
